test/quantization/test_equalize.py

```diff
@@ -0,0 +1,100 @@
+import torch
+import torch.nn as nn
+
+from torch.testing._internal.common_quantization import QuantizationTestCase
+
+import torch.quantization._equalize as _equalize
+
+import copy
+
+class TestEqualizeEager(QuantizationTestCase):
+    def channels_equalized_test(self, tensor1, tensor2, output_axis, input_axis):
```

Comment:
nit: naming for test should be `test_xxx`

Reply:
naming for helper function is camelcase I think, e.g. checkChannelsEqualized

Reply:
im using this function as a helper function for the other defined tests, if you name it test_xxx wouldn't the unittester think its a test function?

Reply:
please see my second comment